research-app/src/App.vue

```diff
@@ -286,12 +297,32 @@
                 </div>
               </div>
             </template>
+            
+            <template v-else-if="currentTool == 'save-state'">
+              <div class="save-state-container">
+                <label class="save-state-title">The current app state can be restored using:</label>
```

Comment:
The descriptive text here should track the change in the tool description, above.